pkg/api/addons.go

```diff
@@ -169,11 +179,11 @@ func (cs *ContainerService) setAddonsConfig() {
 		Containers: []KubernetesContainerSpec{
 			{
 				Name:           "omsagent",
-				Image:          "microsoft/oms:ciprod10162018-2",
 				CPURequests:    "50m",
 				MemoryRequests: "200Mi",
 				CPULimits:      "150m",
 				MemoryLimits:   "750Mi",
+				Image:          "microsoft/oms:ciprod10162018-2",
```

Comment:
can this be changed to not be hardcoded anymore using `ContainerMonitoringAddonName` in k8sComponents? I see you already did half the work by changing the name to match the image

Reply:
punt on this while omsagent team is developing it